tests/ssl-opt.sh

```diff
@@ -688,9 +688,23 @@ requires_openssl_3_x() {
     fi
 }
 
+# skip next test if openssl does not support ffdh keys
+requires_openssl_tls1_3_with_ffdh() {
+    requires_openssl_3_x
+}
+
+# skip next test if it cannot handle ephemeral key exchange
```

Comment:
Minor: if openssl cannot ("it" only makes sense just after reading the previous comment)